src/services/email.service.ts

```diff
@@ -0,0 +1,42 @@
+import * as SendGridMail from '@sendgrid/mail';
+import * as dotenv from 'dotenv';
+
+dotenv.config();
+SendGridMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+/* Send an email to one or more recipients */
+const _sendEmails = async (
```

Comment:
single Email?

Reply:
Old comment before I changed it from sendAnEmail to _sendEmails lol sorry

Reply:
sorry - I meant it would be more useful to expose the single email func